scripts/tasks/publish.ts

```diff
@@ -14,7 +14,7 @@ export const publish: Task = {
   },
   async run({ codeDir }, { dryRun, debug }) {
     return exec(
-      'yarn local-registry --publish',
+      'ts-node --swc --project=../scripts/tsconfig.json ../scripts/run-registry.ts',
```

Comment:
Any particular reason to change the yarn command to this?  before many places were just referring to the same script, now there are 5 different spots with a copy-paste of this line which might make it easy to miss one of them if we need to maintain this 🤔

Reply:
I don't see the benefit TBH, we either have
- `'yarn local-registry --publish'` in multiple places or 
- `'ts-node --swc --project=../scripts/tsconfig.json ../scripts/run-registry.ts'`
... in multiple places.

I figured have 1 less re-direction would make things easier.

Having 1 less level of node processes nesting into another would be good for perf.

Do you recon we'll 
- need to update/change this code often enough 
- it's likely enough we'll forget to update every reference
- and likely enough that this will go unnoticed for long?

...for us to optimize for maintainability here?

I was going to investigate if the `--project=../scripts/tsconfig.json` part is actually needed, because I suspect it isn't, but didn't want to increase the scope further.

Reply:
But I'll revert this, because I think it's a distraction from what we're trying to do.